webview/src/experiments/components/table/styles.module.scss

```diff
@@ -162,6 +162,7 @@ $bullet-size: calc(var(--design-unit) * 4px);
 }
 
 .queued {
+  color: $row-action-icon-color;
```

Comment:
it's not an action though?

Also, should it be aligned with the plots action and should we use `--checkbox-background`?

Reply:
The clock is an icon inside of the row action block. 

If we use `--checkbox-background` it will be almost invisible on most themes. Example: 

https://user-images.githubusercontent.com/37993418/196083307-f8211702-9cf7-4edb-b701-c2d1f7a5510b.mov

This color matches this icon to the icons shown in the sidebar. I would be happy to change to $checkbox-border if we used the same color for empty stars in https://github.com/iterative/vscode-dvc/pull/2601

Reply:
`--vscode-descriptionForeground` for action icons in the header and for action icon in #2601
`--checkbox-border` + `--checkbox-background` for bullet and checkboxes
`--vscode-icon-foreground` - for chevron action and now for queue icons

Both `--vscode-icon-foreground` and `--vscode-descriptionForeground` are quite stable and more or less high contrast as far as I remember. We can probably use one of those only for all icons (stars and the header and chevron). Probably I picked the `--vscode-descriptionForeground` for stars since it had less contrast compared to the surrounding checkboxes and bullets. Can we use it for chevron - don't know, need to try.

For queue, I still make sense to use checkboxes (the border one if background is not visible). Otherwise it'll might stand out too much from bullets and checkboxes and stars also. `--vscode-icon-foreground` had the highest contrast of three options as far as I remember.

Reply:
Using `--vscode-icon-color` for all icons:

https://user-images.githubusercontent.com/37993418/196090395-67983a1e-a2bb-4abe-8ebd-a2d1605569b0.mov

Using `--checkbox-border`

https://user-images.githubusercontent.com/37993418/196091286-85463add-69d2-447e-8b2f-f97b55ab0d46.mov

Reply:
How about `--vscode-descriptionForeground`, the current one for actions in the header ? (clearly `--checkbox-border` is not visible enough, that's why I didn't pick if for stars).

Reply:
The thing that bothers me is the discrepancy with the tree. We could change the circle color to match quite easily but the clock would be more difficult:

https://user-images.githubusercontent.com/37993418/196102189-233697cc-fec1-4c60-8538-4effff2d565f.mov

Reply:
Thanks, Matt. Look reasonable to me. Checkboxes, stars, bullets blend nicely. WDYT?

I'm less worried about the tree tbh. And to be fair, we have a very large discrepancy with the checkboxes, with the action icons, etc (those things in the table that drive us towards _not_ using the `--vscode-icon-color` in the first place). We should then update those? We'll face some other issues - e.g. in some themes webview has a different background / style compared to the side panel.

Btw, on this screenshot:

<img width="146" alt="Screen Shot 2022-10-17 at 9 58 40 AM" src="https://user-images.githubusercontent.com/3659196/196238876-0b087f69-478d-400a-ba47-af8e94047a45.png">

there are a few issues (I can find and fix them later on my own if it takes time) - plots icon color is wrong and chevron ideally should also have the same color as the stars / checkboxes / other icons (so that we can get rid of extra styles at least).

Reply:
> Thanks, Matt. Look reasonable to me. Checkboxes, stars, bullets blend nicely. WDYT?

I keep coming back to [this comment](https://github.com/iterative/vscode-dvc/pull/2601#discussion_r996490558):

> I think the star action itself is not as visible enough then. I admit that I like it visually more, but I think we are sacrificing usability to aesthetics in that case.

I think we should use the icon color for all the icons. We can use that horrible yellow for the star to make it more obvious when it is selected.

Reply:
> I think we should use the icon color for all the icons. We can use that horrible yellow for the star to make it more obvious when it is selected.

Just to clarify that I understand you correctly. so, in your green theme example - all of the object will become bright white - bullets, actions in the header, chevron, queue icons ... it feels a bit aggressive for the table to me (since it's not a small set of icons, but rather a repeating pattern per row) + checkboxes will in the complete shadow because of this.

I'm fine with the the way it looks with the `--vscode-descriptionForeground` . Let's proceed with this please then.

Reply:
Have updated to this:

![Screen Shot 2022-10-18 at 8 46 38 am (2)](https://user-images.githubusercontent.com/37993418/196290097-c1f106de-5de7-47f4-92a0-db893badbefb.png)


Will update the PR description, merge this and close #2601 as this contains that change as well.